script-state.json

```diff
@@ -20,7 +28,25 @@
   "panelScoreVisibility": "Invisible",
   "panelTimePassesVisibility": "Invisible",
   "panelTurnOrderVisibility": "Invisible",
+  "playerBlocks": {
+    "Blue": "36bbcc",
+    "Green": "fac8e4",
+    "Orange": "6b5b4b",
+    "Purple": "661aa3",
+    "Red": "c68e2c",
+    "Yellow": "c3c59b"
+  },
+  "playerTables": {
```

Comment:
`playerTables` doesn't need to be in save state anymore right?

Reply:
Good catch.  I don't typically pay attention to script-state.json since, well, I'm still of the mind we should just delete script-state on releases and the mod should be able to rebuild it.

Fixed.